lib/irc.js

```
@@ -956,7 +956,19 @@ Client.prototype.notice = function(target, text) {
 
 Client.prototype._speak = function(kind, target, text) {
     var self = this;
-    var maxLength = this.maxLineLength - target.length;
```

Fail early, IMO. We should fail as soon as we know the target is invalid.